### PR TITLE
style: update deactivated state styles in onboarding button

### LINKED ISSUE #952: Update desactivated state of onboarding button
# Current style

Image

# Desired style

Remove box-shadow (on the deactivated state only) and change the background to "secondary"
Border color should be "Transparent lighter"

Image

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=3193-63417&mode=design&t=sj4NhD0g7qMJY0yH-4

### PR DESCRIPTION
**Pull request by indrajitbnikam**
closes #952 

Update the styles as per specs mentioned in #952 

**Before change** 👇
image

**After change** 👇
image

### CODE REVIEW

**Comment by indrajitbnikam** [comment]
Hi @FelixMalfait @Bonapara  , Let me know if I made the change in the correct place.

File: front/src/modules/ui/button/components/MainButton.tsx [hunk 1]
```diff
@@ -31,9 +31,22 @@ const StyledButton = styled.button<Pick<Props, 'fullWidth' | 'variant'>>`
         return theme.background.primary;
     }
   }};
-  border: 1px solid ${({ theme }) => theme.border.color.light};
+  border: 1px solid;
+  border-color: ${({ theme, disabled }) => {
+    if (disabled) {
+      return theme.background.transparent.light;
```
Comment (charlesBochet) [inline front/src/modules/ui/button/components/MainButton.tsx:37]: transparent.lighter

Comment (indrajitbnikam) [reply front/src/modules/ui/button/components/MainButton.tsx]: Done 👍

**Review by charlesBochet — APPROVED** [review]
Thank you!

**Pull request merged**